src/common/user-store.ts

```diff
@@ -28,7 +28,8 @@ export interface UserPreferences {
   downloadBinariesPath?: string;
   kubectlBinariesPath?: string;
   openAtLogin?: boolean;
-  hiddenTableColumns?: Record<string, string[]>
```

Comment:
Please rebase correctly, hiding tables was added in a different PR.

Reply:
I merged hidden columns and column sizes into storeTableConfig, every column in this config has 2 properties, visible and width. Also, I've added index column, now it is just for simplifying the search of the columns, but it could be useful for the future ( I would like to add the ability to change the order of columns)